tests/python/frontend/tflite/test_forward.py

```diff
@@ -512,6 +563,21 @@ def test_forward_inception_v3_net():
     tvm.testing.assert_allclose(np.squeeze(tvm_output[0]), np.squeeze(tflite_output[0]),
                                 rtol=1e-5, atol=1e-5)
 
+def test_forward_inception_v4_net():
+    '''test inception v4 tflite model'''
```

Comment:
```suggestion
    """Test the Inception v4 TF Lite model."""
```

Reply:
Do I need to fix the comment style for all other end-to-end tests such as """Test the Mobilenet V1 TF Lite model.""" instead of '''test mobilenet v1 tflite model'''?